plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -58,6 +58,7 @@ module UntypedPlutusCore.Evaluation.Machine.Cek.Internal
     , Slippage
     , defaultSlippage
     , NTerm
+    , runCekM
```

Comment:
is this used somewhere? I guess it must be.

Reply:
It is used by the steppable cek to provide the same `runCekDeBruijn` run-to-completion interface as the original cek